docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -46,7 +46,9 @@ The file represents a collection of JSON elements describing known issues.
         "stepPattern": "Then `1` is = `2`.+", <6>
         "variablePatterns": { <7>
             "var":  "value-.*"
-        }
+        },
+        "failTestCaseFast": true, <8>
+        "failTestSuiteFast": true <9>
```

Comment:
I think we should follow BDD-like approach:
 - `failScenarioFast`
 - `failStoryFast`